.gitignore

```diff
@@ -136,4 +136,10 @@ dmypy.json
 .pyre/
 
 # VS Code settings
-.vscode/
\ No newline at end of file
+.vscode/
+
+# other
+modded_files
```

Comment:
Is it easier to have these other files here instead of using local git branches or keeping a working directory outside of the repo? I just ask because when we add these to the `.gitignore` we can't use them for other purposes